### DISCUSSION TITLE
Vieweing Log:debug logs

### DISCUSSION THREAD

**Discussion by sa-mendez** [LunarVim/LunarVim#3615] [Q&A] [answered]
I am trying to diagnose an issue with getting one of my language servers to load up, and noticed there are a bunch of Log:debug statements throughout the code that would be useful for this.  However, I cannot find where these logs ultimately get output to, or if there is a way for me to turn them on.

Can anybody point me in the right direction here?

Thank you.

**Comment by pgierz** [comment, marked as answer]
For logs, I have previously used:
```
<leader> >> +LunarVim >> +logs >> debug
```

On the default configuration, that should map to

<kbd>Space</kbd>  <kbd>L</kbd> <kbd>l</kbd> <kbd>d</kbd>

**Reply by sa-mendez** [reply]
Thanks,  that helped a bit in tracking down what I needed to change.  In fact, I should have seen it.

The setting is right there in my config.  I changed:

`lvim.log.level = "warn"`

to 

`lvim.log.level="debug"`

And then I was able to look at my lvim.log file to see the new logs.
